codewof/programming/content/en/driver-speed/question.md

```diff
@@ -0,0 +1,10 @@
+# Driver speed
+
+Write a function `check_speed(speed)` that checks if a driver is driving above the speed limit.
+
+- If the speed is less than or equal to 50, the function should **print** `Ok.`
+- If the speed is above 50, the function should **print** `Too fast!`
+- If the speed is 75 or higher then the function should *also* **print** `License suspended.`
+
+For example: `check_speed(53)` should **print** `Too fast!`, and
+`check_speed(70)` should **print** `Too fast! License suspended.`
```

Comment:
This is the text `Too fast! License suspended.` (with a space) but your code checks for `Too fast!\nLicense suspended.` (newline)